packages/cinterop/build.gradle.kts

```diff
@@ -131,7 +160,7 @@ kotlin {
             cinterops.create("realm_wrapper") {
                 defFile = project.file("src/nativeCommon/realm.def")
                 packageName = "realm_wrapper"
-                includeDirs(project.file("$relativeCorePath/src/"))
+                includeDirs("$absoluteCorePath/src/")
```

Comment:
Have you verified that this still allows compilation from both inside `packages` and `test`. As far as I recall there was some issue with that.